src/plotman/manager.py

```diff
@@ -121,29 +121,38 @@ def key(key: str) -> job.Phase:
 
             log_file_path = log_cfg.create_plot_log_path(time=pendulum.now())
 
-            plot_args: typing.List[str] = ['chia', 'plots', 'create',
+            if plotting_cfg.type == "madmax":
+                plot_args: typing.List[str] = ['chia_plot',
+                    '-n', str(1),
+                    '-r', str(plotting_cfg.n_threads),
+                    '-u', str(plotting_cfg.n_buckets),
+                    '-t', tmpdir if tmpdir.endswith('/') else (tmpdir + '/'),
+                    '-d', dstdir if dstdir.endswith('/') else (dstdir + '/') ]
+            else:
+                plot_args: typing.List[str] = ['chia', 'plots', 'create',
```

Comment:
Or, it may not be needed at all.

```suggestion
            plot_args: typing.List[str]
            if plotting_cfg.type == "madmax":
                plot_args = ['chia_plot',
                    '-n', str(1),
                    '-r', str(plotting_cfg.n_threads),
                    '-u', str(plotting_cfg.n_buckets),
                    '-t', tmpdir if tmpdir.endswith('/') else (tmpdir + '/'),
                    '-d', dstdir if dstdir.endswith('/') else (dstdir + '/') ]
            else:
                plot_args = ['chia', 'plots', 'create',
```